### ISSUE TITLE
[class] can't convert to class when overriding apply method

### ISSUE THREAD

**Issue by e-cloud** [lebab/lebab#194]
like
```
function AutomaticPrefetchPlugin() {
}
module.exports = AutomaticPrefetchPlugin;
AutomaticPrefetchPlugin.prototype.apply = function(compiler) {
};
```

class transform has no effect

**Comment by nene** [comment]
Unable to reproduce, for me the class transform produces the following:

```js
class AutomaticPrefetchPlugin {
  apply(compiler) {
  }
}

module.exports = AutomaticPrefetchPlugin;
```

**Comment by e-cloud** [comment]
Tried a bunch of cases, finally i found out when using lebab combining `arrow` and `class`, `arrow` should behind `class`, otherwise in some cases like the case above, `class` transform has no effect.

I haven't figout why yet.

**Comment by nene** [comment]
Ah, indeed.

- `arrow` does not convert functions that use `this`, so usually it ends up skipping class methods.
- `class` does not convert arrow functions, because they could bind `this` from outer scope.

The fix is for `class` transform to check whether arrow function uses `this` - if it does not, it can be transformed to method without worries. Then the transforms should work in either order.

Thanks for the bug report!

**Comment by nene** [comment]
Fixed in 2.6.1
